<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlPanel</class>
 <widget class="QMainWindow" name="ControlPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ControlPanel</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="0">
     <widget class="QPushButton" name="Button7">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #ada397;
	border: 1px solid gray;
	padding: 6px;
}

QPushButton:pressed {
  	background-color: #bdcebe;
	border: 1px solid gray;
	padding: 6px;
}
</string>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="Button5">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #ada397;
	border: 1px solid gray;
	padding: 6px;
}

QPushButton:pressed {
  	background-color: #bdcebe;
	border: 1px solid gray;
	padding: 6px;
}
</string>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QPushButton" name="Button0">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #ada397;
	border: 1px solid gray;
	padding: 6px;
}

QPushButton:pressed {
  	background-color: #bdcebe;
	border: 1px solid gray;
	padding: 6px;
}
</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QPushButton" name="Button6">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #ada397;
	border: 1px solid gray;
	padding: 6px;
}

QPushButton:pressed {
  	background-color: #bdcebe;
	border: 1px solid gray;
	padding: 6px;
}
</string>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="Activate">
      <property name="text">
       <string>Activate</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QLineEdit" name="Display">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Andale Mono</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	background-color: #3b3a30;
	border: 1px solid gray;
	color: #82b74b;
}</string>
      </property>
      <property name="text">
       <string>Input Passcode</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="Button9">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #ada397;
	border: 1px solid gray;
	padding: 6px;
}

QPushButton:pressed {
  	background-color: #bdcebe;
	border: 1px solid gray;
	padding: 6px;
}
</string>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="Deactivate">
      <property name="text">
       <string>Deactivate</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="Button8">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #ada397;
	border: 1px solid gray;
	padding: 6px;
}

QPushButton:pressed {
  	background-color: #bdcebe;
	border: 1px solid gray;
	padding: 6px;
}
</string>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QPushButton" name="Button3">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #ada397;
	border: 1px solid gray;
	padding: 6px;
}

QPushButton:pressed {
  	background-color: #bdcebe;
	border: 1px solid gray;
	padding: 6px;
}
</string>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="Button4">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #ada397;
	border: 1px solid gray;
	padding: 6px;
}

QPushButton:pressed {
  	background-color: #bdcebe;
	border: 1px solid gray;
	padding: 6px;
}
</string>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="CurrentState">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QPushButton" name="Button2">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #ada397;
	border: 1px solid gray;
	padding: 6px;
}

QPushButton:pressed {
  	background-color: #bdcebe;
	border: 1px solid gray;
	padding: 6px;
}
</string>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QPushButton" name="ChangePass">
      <property name="text">
       <string>Change pass</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QPushButton" name="Button1">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #ada397;
	border: 1px solid gray;
	padding: 6px;
}

QPushButton:pressed {
  	background-color: #bdcebe;
	border: 1px solid gray;
	padding: 6px;
}
</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QPushButton" name="Ok">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QPushButton" name="StopAlarm">
      <property name="text">
       <string>Stop Alarm</string>
      </property>
     </widget>
    </item>
    <item row="8" column="2">
     <widget class="QPushButton" name="SendReport">
      <property name="text">
       <string>Send Report</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>391</width>
     <height>37</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
